### ISSUE TITLE
[1.0.20] Light Doesn't work as intended

### ISSUE THREAD

**Issue by WaterKing201030** [AllTheMods/ATM-8#1481]
I found that the light level calculation doesn't work as intended in 1.0.20.
Here is what my mob farm looks like: it was a box built with tinted glass and stone, and a door using Mobfilter from dark utilites.
![2023-06-20_20 06 58](https://github.com/AllTheMods/ATM-8/assets/73704964/77c9bf4a-e923-4296-bedb-271035597955)
However at the far corner of the box the light is larger than 7, but it should be 0. 
![2023-06-20_20 10 18](https://github.com/AllTheMods/ATM-8/assets/73704964/5af97e46-81ba-491f-82ec-49355f7404af)
Here is a world which also faces the problem.
[testworld.zip](https://github.com/AllTheMods/ATM-8/files/11800499/testworld.zip)
I re-enter the world many times and found that this issue may appeared randomly. For some reason I don't know it have been repaired.
![2023-06-20_21 02 21](https://github.com/AllTheMods/ATM-8/assets/73704964/42d33ba1-25e6-4454-b765-595cc831e40f)

**Comment by Niklars** [comment]
I have same issue

**Comment by BalloS-Twok** [comment]
Here is the same.
My mobfarm is completly disabled and I have no idea to get rid of the light.

**Comment by MrChunks** [comment]
I also had this problem after upgrading from 1.0.19b to both 20 and 21b.
My mob farm was based on Dreadful Dirt, and the dirt burned (exposed to daylight) after I broke the Vector Plate (dark utilities) that was sitting on it, even though I know it was under a roof.

**Comment by DugganB** [comment]
Same issue here using `1.0.21b`. Have not found a workaround.

**Comment by MrChunks** [comment]
Any idea if it's tied to upgrading from a previous version? Or does it work properly if you create a new cursed earth mob farm after you're running 1.0.21b already?

**Comment by DugganB** [comment]
@MrChunks No it doesnt have to do with upgrading, I made this world on `1.0.21b` from the start. My farm intersects the chunk boundary, going to see if making a new farm not on chunk boundary could fix it.

**Comment by MrChunks** [comment]
Gotcha. I was also wondering if it had anything to do with the special tinted glass. I shoulda tested more before I rolled back.

**Comment by FloryMafia** [comment]
im also having this issue, i tried to use dreadful dirt as a substitute instead however it seems that solution was only temporary as its now completely stopped i assume the dirt has now been burned (i havnt checked because the mob spawner is surrounded by a aquarium

**Comment by scotty9090** [comment]
Based on my testing, I believe this issue is at least in part related to Starlight (unsurprisingly given that it's optimizing the lighting engine.)

On the prior version of the pack, I was experiencing the dark spot bug with Radon, so I swapped it out for Starlight and instantly began having a bright mob farm.

That has continued in this version of the pack.  Sometimes I've been able to solve this by filling every block inside my farm with dirt/cobble and then vein mining it out, but not always or not always completely.

I've since removed Starlight and so far I haven't encountered the issue.
